men_team_1/spec/models/pairing_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Pairing, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Not a great test. ;) Generally you wouldn't commit "TODO"-style comments to master, but because this is a prototype project it's not totally unreasonable. Proceed with care.

Reply:
It's probably best to remove pending lines for now.